templates/server-configmap.yaml

```diff
@@ -77,8 +77,10 @@ data:
                 url:
                     scheme: "{{ $.Values.elasticsearch.scheme }}"
                     host: "{{ $.Values.elasticsearch.host }}:{{ $.Values.elasticsearch.port }}"
+                username: "{{ $.Values.elasticsearch.username }}"
```

Comment:
does this still work if the ES endpoint doesn't require auth?

Reply:
Good question. Yes, because the ES client library checks for empty string on username/password